### DISCUSSION TITLE
Adding updates over Winget (Window Package Manager)

### DISCUSSION THREAD

**Discussion by arfx** [woelper/oculante#310] [Ideas]
Hey I just noticed that I'm way behind the current version. 
Can you add your tool to winget to see when an update is available and install this with the Window Package Manager.

https://learn.microsoft.com/en-us/windows/package-manager/package/

**Comment by woelper** [comment]
Hi! Thanks for this idea. There is an update button in Oculante - does this work for you?
Oculante is a "portable" app, so there is nothing to install. The docs above suggest that an installer is needed, which I don't have currently. This would make this task a lot harder. If you like, you could maybe into generating the required yaml file from the Cargo.toml as I think this process should be automated.
